app/jobs/sitemap_indexer_job.rb

```diff
@@ -2,6 +2,6 @@ class SitemapIndexerJob < ActiveJob::Base
   queue_as :searchgov
 
   def perform(searchgov_domain:)
-    searchgov_domain.index_sitemap
+    SitemapIndexer.new(searchgov_domain: searchgov_domain).index
```

Comment:
@noremmie , you and I talked about this change a few days ago; the SearchgovDomain#index_sitemap method now enqueues a SitemapIndexerJob.